packages/teams-js/prepBetaRelease.js

```diff
@@ -128,37 +128,46 @@ function getNewNextDevSuffixNum(currNextDevVer, currPkgJsonVer) {
     // If the next-dev beta suffix is not equal to or exactly 1 higher than the package.json beta suffix, throw an error since this does not follow our versioning structure.
     // e.g. package.json version: 2.0.0-beta.1 and current next-dev version: 2.0.0-beta.0-dev.0 should throw an error.
   } else if (betaSuffixNumInCurrNextDev !== betaSuffixNumInPkgJsonVer) {
-    throw new Error(`Invalid beta version suffix number ${betaSuffixNumInPkgJsonVer} in package.json version ${currPkgJsonVer}`);
```

Comment:
Sorry I missed this earlier. Can you please update the `getNewNextDevSuffixNum` function to `getNewBetaSuffixNum` and any references to "*nextDev*"? E.g. `currNextDevVer` in the function signature. I also think that code should be simplified now that we're not looking for "*.beta.x-dev.y".

Reply:
Ah upon further inspection, I see this function isn't being used at all. Which I think is a problem (see my comment on line 161).

Reply:
I think it would be great if you could clean up the script and remove all unused functions/variables as part of this change.